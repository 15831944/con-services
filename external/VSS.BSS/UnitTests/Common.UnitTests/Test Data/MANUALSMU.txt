<?xml version="1.0" encoding="UTF-8"?>
<manualSMU:topic xmlns:manualSMU="http://www.cat.com/em/feed/v1/ManualServiceMeter" xmlns:feed="http://www.cat.com/em/feed/v1/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v1/ManualServiceMeter https://emfeed.rd.cat.com/EMFeedFiles/schema/v1/ManualServiceMeter.xsd">   
<nextBuffer>
   <url>https://emfeed.rd.cat.com/EMFeedService/v1/feed/allqueue/MANUALSMU/28525629</url>
   <moreData>false</moreData>
</nextBuffer>
<message>
      <header>
         <messageId>28525606</messageId>
         <masterMsgId>8172080</masterMsgId>
         <equipment>
            <serialNumber>HORSTMANN20091110</serialNumber>
            <make>IHC</make>
         </equipment>
         <moduleCode>HORSTMANN_PHILLIP_A</moduleCode>
         <moduleTime>2009-07-28 10:04:01</moduleTime>
         <receivedTime>2009-12-09 21:27:25</receivedTime>
      </header>
      <serviceMeter>
         <before>
            <value>5000</value>
            <uom>Hours</uom>
         </before>
         <after>
            <value>1100</value>
            <uom>Hours</uom>
         </after>
      </serviceMeter>
   </message>
<message>
      <header>
         <messageId>28525629</messageId>
         <masterMsgId>8172097</masterMsgId>
         <equipment>
            <serialNumber>HORSTMANN20091110</serialNumber>
            <make>IHC</make>
         </equipment>
         <moduleCode>HORSTMANN_PHILLIP_A</moduleCode>
         <moduleTime>2008-09-18 13:08:45</moduleTime>
         <receivedTime>2009-12-09 21:29:44</receivedTime>
      </header>
      <serviceMeter>
         <before>
            <value>50</value>
            <uom>Hours</uom>
         </before>
         <after>
            <value>200</value>
            <uom>Hours</uom>
         </after>
      </serviceMeter>
 </message>
 <!--INVALID - Missing the 'Before' hours value -->
 <message>
      <header>
         <messageId>28525729</messageId>
         <masterMsgId>8172297</masterMsgId>
         <equipment>
            <serialNumber>HORSTMANN20091110</serialNumber>
            <make>IHC</make>
         </equipment>
         <moduleCode>HORSTMANN_PHILLIP_A</moduleCode>
         <moduleTime>2008-09-18 15:08:45</moduleTime>
         <receivedTime>2009-12-09 21:35:44</receivedTime>
      </header>
      <serviceMeter>
         <before>            
            <uom>Hours</uom>
         </before>
         <after>
            <value>200</value>
            <uom>Hours</uom>
         </after>
      </serviceMeter>
 </message>
 <!--INVALID - Missing the 'After' section -->
 <message>
      <header>
         <messageId>28525729</messageId>
         <masterMsgId>8172297</masterMsgId>
         <equipment>
            <serialNumber>HORSTMANN20091110</serialNumber>
            <make>IHC</make>
         </equipment>
         <moduleCode>HORSTMANN_PHILLIP_A</moduleCode>
         <moduleTime>2008-09-18 15:08:45</moduleTime>
         <receivedTime>2009-12-09 21:35:44</receivedTime>
      </header>
      <serviceMeter>
         <before>            
			<value>50</value>
            <uom>Hours</uom>
         </before>
      </serviceMeter>
 </message>
</manualSMU:topic>