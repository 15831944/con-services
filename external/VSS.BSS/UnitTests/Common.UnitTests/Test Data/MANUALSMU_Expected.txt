<?xml version="1.0" encoding="utf-8"?><ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10"><NHDataWrapper><Data z:Id="i1" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataServiceMeterAdjustment" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>0</d3p1:AssetID><d3p1:DebugRefID>28525606</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2009-07-28T10:04:01</d3p1:EventUTC><d3p1:GPSDeviceID>HORSTMANN_PHILLIP_A</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:RuntimeAfterHours>1100</d3p1:RuntimeAfterHours><d3p1:RuntimeBeforeHours>5000</d3p1:RuntimeBeforeHours><d3p1:SourceMsgID>8172080</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i2" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataServiceMeterAdjustment" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>0</d3p1:AssetID><d3p1:DebugRefID>28525629</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:DimSource i:nil="true" /><d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses"><d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" /></d3p1:DimSourceReference><d3p1:EventUTC>2008-09-18T13:08:45</d3p1:EventUTC><d3p1:GPSDeviceID>HORSTMANN_PHILLIP_A</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:RuntimeAfterHours>200</d3p1:RuntimeAfterHours><d3p1:RuntimeBeforeHours>50</d3p1:RuntimeBeforeHours><d3p1:SourceMsgID>8172097</d3p1:SourceMsgID><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper></ArrayOfNHDataWrapper>